src/backend/access/bitmap/bitmapinsert.c

```diff
@@ -2145,8 +2145,7 @@ insertsetbit(Relation rel, BlockNumber lovBlock, OffsetNumber lovOffset,
 		MemSet(buf->cwords, 0,
 				buf->num_cwords * sizeof(BM_HRL_WORD));
 	}
-	MemSet(buf->hwords, 0,
-		   BM_CALC_H_WORDS(buf->num_cwords) * sizeof(BM_HRL_WORD));
+	MemSet(buf->hwords, 0, sizeof(buf->hwords));
```

Comment:
What about removing MemSets, because the function is called from one place, where buf has already been cleaned?